src/pages/publication/contributors_tab/components/CreateContributorModalContent.tsx

```diff
@@ -21,42 +24,70 @@ const StyledDescription = styled(NormalText)`
   white-space: pre-wrap;
 `;
 
-const CreateContributorModalContent: FC = () => {
+const StyledProgressContainer = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 36rem;
+  height: 16rem;
+  padding: 2rem;
+`;
+
+interface CreateContributorModalContentProps {
+  addAuthor: (author: Authority) => void;
+  handleCloseModal: () => void;
+}
+
+const CreateContributorModalContent: FC<CreateContributorModalContentProps> = ({ addAuthor, handleCloseModal }) => {
   const [readMore, setReadMore] = useState(false);
+  const [loading, setLoading] = useState(false);
   const { t } = useTranslation('common');
 
   const toggleReadMore = () => setReadMore(!readMore);
 
+  const handleSubmit = async (values: FormikValues) => {
+    setLoading(true);
+    const createdAuthority = await createAuthority(values.firstName, values.lastName);
+    addAuthor(createdAuthority);
+    handleCloseModal();
```

Comment:
snackbar som bekrefter opprettelse?

Reply:
tja, du ser jo at den forfatteren får en check mark da, så vet ikke om det er nødvendig? men kanskje lurt å heller ha for mye enn for lite info?

Reply:
åjaa, man legges til som contributor samtidig ja. Trodde man måtte gjøre det manuelt i neste steg igjen. Da er det sikkert greit som det er ja!